dev/add-to-output-repos.sh

```diff
@@ -0,0 +1,34 @@
+branch=stable
+if [ "$1" = "beta" ]; then
+  branch=master
+fi
+
+EMBERVERSION=`ember version | grep "ember-cli:" | cut -d' ' -f2`
+
+commands=(new addon)
+
+for i in ${commands[@]}; do
+  command=$i
+  repo_folder=ember-$command-output
+
+  local_folder=my-$command
```

Comment:
Can we make this `/tmp/my-$command` as it's [safely out of the scope](https://github.com/ember-cli/ember-cli/blob/master/.gitignore#L3) of accidental self-trolling via `git`?